processor/src/main/java/org/mapstruct/ap/MappingProcessor.java

```diff
@@ -347,4 +378,15 @@ public int compare(ModelElementProcessor<?, ?> o1, ModelElementProcessor<?, ?> o
             return Integer.compare( o1.getPriority(), o2.getPriority() );
         }
     }
+
+    private static class DeferredMapper {
```

Comment:
I'm not sure. Should we write some javadoc about the purpose of this private static class? Or is it obvious?

Reply:
I thought that it was obvious from the context. If it isn't I can add some javadoc

Reply:
No, you are right